packages/nvidia_nat_weave/src/nat/plugins/weave/fastapi_plugin_worker.py

```
@@ -0,0 +1,120 @@
+        # Add all standard routes first
+        await super().add_routes(app, builder)
+
+        # Add Weave-specific routes
+        await self._add_weave_feedback_route(app, builder)
+
+    async def _add_weave_feedback_route(self, app: FastAPI, builder: WorkflowBuilder):
+        """Add the Weave feedback endpoint if Weave telemetry is configured."""
+
+        # Find Weave telemetry exporter configuration
+        weave_config = None
+        for name, exporter_config in builder._telemetry_exporters.items():
```

Ruff (0.14.4)

Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)